components/Nav.jsx

```diff
@@ -7,7 +7,7 @@ const Nav = () => {
     
     return (
       <nav className="nav">
-        <Link to="/home"> Homepage </Link>
+        <Link to="/home" className="link-homepage"> Homepage </Link>
```

Comment:
I realised I'm viewing your app in dark mode where the link is black on black haha you might need to alter your dark mode styling, but I'll still approve this for now!